buildSrc/version.properties

```diff
@@ -48,3 +48,7 @@ jmh               = 1.26
 # when updating this version, also update :qa:evil-tests
 jimfs = 1.2
 jimfs_guava = 30.1-jre
+
+# test framework
+networknt_json_schema_validator = 1.0.48
+commons_lang3                   = 3.9
```

Comment:
This one is also used in the `ingest-attachment` plugin, so I wonder if it should instead be in the "optional dependencies" section above?